File: solution_set1/2330. Valid Palindrome IV.py
##ss
class Solution:
    def makePalindrome(self, s: str) -> bool:
        
        n = len(s)
        c = 0
        for x in range(n//2):
            if s[x]!=s[-1-x]:
                c+=1
                
        return c in [0,1,2]
        

File: cpp_solutions/2330. Valid Palindrome IV.cpp
//sss
class Solution {
public:
    bool makePalindrome(string s) {
        int change = 0;
        int n = s.length();
        
        for(int x=0; x<floor(n/2);++x) {
            if (s[x]!=s[n-1-x]) {
                change++;
            }
            
        }
        return change<=2;
        
    }
};
